File: programmers_webpage_problems/Level1/약수의 개수와 덧셈/약수의개수와덧셈.py
def prime_count(num):
    count = 0
    for i in range(num):
        if (num) % (i + 1) == 0:
            count += 1
    return count

def solution(left, right):
    answer = 0
    while (left <= right):
        if prime_count(left) % 2 == 0:
            answer += left
        else:
            answer -= left
        left += 1
    
    return answer

File: programmers_webpage_problems/Level1/약수의 개수와 덧셈/약수의개수와덧셈.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int counting(int num)
{
    int i = 0;
    int count = 0;
    
    while (++i <= num)
        if (num % i == 0)
            count++;
    return (count % 2) ? num * -1 : num;
}

int solution(int left, int right)
{
    int answer = 0;
    
    for (int i = left; i <= right; i++)
        answer += counting(i);
    return (answer);
}
